parachain/pallets/eth-app/src/lib.rs

```diff
@@ -85,7 +87,7 @@ decl_module! {
 		// Users should burn their holdings to release funds on the Ethereum side
 		// TODO: Calculate weights
 		#[weight = 0]
-		pub fn burn(origin, recipient: H160, amount: U256) -> DispatchResult {
+		pub fn burn(origin, channel_id: ChannelId, recipient: H160, amount: U256, ) -> DispatchResult {
```

Comment:
actually users will Specify a more higher-level `DeliveryOptions` parameter instead of channel_id

Reply:
lets just keep it simple and get it working and we can improve things after :)

Reply:
fine with me :+1: